src/commands/exit.ts

```diff
@@ -0,0 +1,10 @@
+import { ExitExecuteResult } from "../result.ts";
+
+export function exitCommand(args: string[]): ExitExecuteResult {
+  const code = parseInt(args[0], 10);
```

Comment:
Also, there's some more logic we'll have to do with this: https://github.com/denoland/deno_task_shell/blob/18324b91faa83f94d84ac0c97766a08811d9c4ef/src/shell/commands/exit.rs#L26